tests/unit/AgmLocationChg.spec.ts

```diff
@@ -144,7 +145,7 @@ describe('AGM Location Chg view', () => {
 
     const wrapper = shallowMount(AgmLocationChg, { mocks: { $route, $router } })
     wrapper.vm.$data.dataLoaded = true
-    await Vue.nextTick()
+    await waitForUpdate(2)
```

Comment:
This looks something new :)

Reply:
This function existed before but we rarely use it! It's found in the tests for PDF Upload for example when we need to wait for DOM updates.

The other way to fix it is have 2 `await Vue.nextTick()`. This function takes place instead of having these duplicate lines.